package.json

```diff
@@ -1,10 +1,11 @@
 {
   "name": "hubot",
-  "version": "0.0.0",
+  "version": "1.0.0",
```

Comment:
Going 1.0 seems like a big step to me. How about applying the suggestions postulated in [Semantic Versioning 2.0.0](http://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase), e.g. use version 0.1.0 for this PR?